app/helpers/events_helper.rb

```diff
@@ -0,0 +1,34 @@
+module EventsHelper
+  def modify_event_from_admin_form(params)
```

Comment:
What is this doing?

Reply:
Replaced by the scaffold event logic, removed but leaving helper however as I will use the class